test/integration/targets/test_uri/tasks/main.yml

```diff
@@ -0,0 +1,26 @@
+- name: Assert url tests
+  assert:
+    that:
+      - "'http://google.com' is url"      # simple case
+      - "'htp://google.com' is not url"   # bad schema
+      - "'ftp://google.com' is url"       # other schema
+      - "'file://etc/hosts' is not url"   # uris not urls
+      - "'mailto://me@nowhere.com' is not url"
```

Comment:
I'm pretty sure that the two above are also URLs. (They have a protocol.)

Reply:
see comment above